Service/Request/RequestFactory.php

```diff
@@ -22,6 +23,8 @@ class RequestFactory implements RequestFactoryInterface
 {
     use LoggerAwareTrait;
 
+    const METHODS_WITHOUT_BODY = ['GET', 'HEAD', 'OPTIONS'];
```

Comment:
We can add the `private` modifier since we support PHP 7.3 and above

Reply:
Fixed